oidc-wp/openid-connect-generic
# Fixes QS not being added to the redirect URL

### All Submissions:






### Changes proposed in this Pull Request:

On my WP installation the QS is lost when saving the redirect URL. Those changes fix the issue.

### How to test the changes in this Pull Request:

1. go to any login-protected URL with any query string
2. login
3. you should get back to page with query string present

### Other information:




### Changelog entry

> Fixes QS not being added to the redirect URL
